src/pages/academy/components/utility.ts

```diff
@@ -33,3 +34,28 @@ export const addScriptForCIO = (is_eu) => {
 export const handleTag = (tag_name) => {
     navigate(`/academy/search?category=${encodeURI(`${tag_name}`)}`)
 }
+
+export const dataFilter = (data) => {
+    const extra = {
+        smart_trader: 'c1925a11-e999-4750-97af-cc243186ce40',
+        options: '1d194e63-e40e-440f-85f1-79e9030e2114',
+    }
```

Comment:
Why do we need to manually filter these? Are the text included in the blog image itself?

Reply:
I just manually found it and include in array. Have no acces using `extra.includes(item)` that's why decided filter it manually

Reply:
Ok now I got it. Changed filtration by blog title instead of just id

Reply:
With Mohammad we changed property visibility inside CMS and everything works fine without it